ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh

```diff
@@ -434,5 +434,8 @@ echo "${BASTION_HOST_PUBLIC_DNS}" > "${SHARED_DIR}/bastion_dns"
 PROXY_URL="http://${PROXY_CREDENTIAL}@${BASTION_HOST_PUBLIC_DNS}:3128"
 echo "${PROXY_URL}" > "${SHARED_DIR}/public_proxy_url"
 
+MIRROR_REGISTRY_URL="${BASTION_HOST_PUBLIC_DNS}:5000"
+echo "${MIRROR_REGISTRY_URL}" > "${SHARED_DIR}/mirrorregistryhost"
```

Comment:
How to pass down the mirror registry credential for the following steps? read them from vault ?

Reply:
yes, mount `openshift-custom-mirror-registry` while doing mirror action https://github.com/openshift/release/pull/24888/files#diff-798d9c68486837f9b3b860e28ce7c9581b060a27e55d60f2c77a0a3d9eaf27a9R12-R14